.github/workflows/ci_cpu.yml

```diff
@@ -36,8 +36,12 @@ jobs:
         run: |
           python ts_scripts/install_dependencies.py --environment=dev
       - name: Torchserve Sanity
-        run: |
-          python torchserve_sanity.py
+        uses: nick-fields/retry@v2
+        with:
+          timeout_minutes: 60
+          max_attempts: 3
```

Comment:
```suggestion
          max_attempts: 3
          retry_on: error
          
```

Reply:
Thanks @msaroufim . This might be the issue